extensions/flyway/runtime/src/main/java/io/quarkus/flyway/runtime/FlywayDataSourceRuntimeConfig.java

```diff
@@ -120,4 +120,16 @@ public static final FlywayDataSourceRuntimeConfig defaultConfig() {
     @ConfigItem(defaultValue = "true")
     public boolean createSchemas;
 
+    /**
+     * Prefix of every placeholder (default: ${ )
```

Comment:
I wonder if we should just define the `defaultValue` so that it's automatically documented (and we would have to initialize the prefix and suffix to these values).

Reply:
As long as we keep the same defaults as Flyway I think it can be useful, otherwise a developer needs to check the extension properties and match them with Flyway properties to discover the default.
This applies to all the Flyway extension configurations